server/data/audit.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
       })
   },
 
-  async getEvents(action, filters = {}, startMoment, endMoment) {
+  async getEvents(action, filters, startMoment, endMoment) {
```

Comment:
new eslint rule doesn't like default args not being the last one - this is only called in one place and always passes through a value